includes/class-monk-links.php

```diff
@@ -499,9 +499,7 @@ public function monk_add_language_author_permalink( $link, $author_id ) {
 		$default_language_url = get_option( 'monk_default_language_url', false );
 		$language             = ( get_query_var( 'lang' ) ) ? get_query_var( 'lang' ) : $this->site_language;
 
-		if ( $this->monk_using_permalinks() ) {
```

Comment:
Since the changes we have made in the home_url() returning value, there is no need to change with the permalinks active